### PR TITLE
Update dependencies +++ version bump

### PR DESCRIPTION
**Pull request by oliverkierepka**
- update @angular-devkit/core to 12.2.18 
- update @angular-devkit/schematics to 12.2.18 
- update @angular/schematics to 12.2.18
- update @phenomnomnominal/tsquery to 5.0.0
- update inquirer ^9.1.4
- update inquirer-autocomplete-prompt to ^3.0.0
- update inquirer-fuzzy-path to ^2.3.0
- update rxjs to ^6.6.7
- update strip-json-comments to ^5.0.0
- update typescript to ^4.0.2
- version bump @schuhard/schematics-core to 0.5.0

### CODE REVIEW

**Comment by oliverkierepka** [comment]
@schuchard I just updated some dependencies and made a version bump
if everything is fine i would be happy to see this released :)

File: packages/schematics-core/src/test-utils.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export async function runSchematic(opt?: {
   collectionPath?: string;
 }): Promise<UnitTestTree> {
   const schematicRunner = new SchematicTestRunner(
-    name,
+    opt?.name || 'schematics',
```
Comment (oliverkierepka) [inline packages/schematics-core/src/test-utils.ts:58]: @schuchard Not sure about this line. It showed an error.. so i fixed it like so.
Tests did work. What do you think?

**Comment by schuchard** [comment]
If you run `yarn bootstrap` and `yarn test:unit`, do the tests pass? It's not passing for me. Also, you pushed us a yarn.lock file referencing your artifactory so I was unable to test with exactly what you had.

**Comment by oliverkierepka** [comment]
> If you run `yarn bootstrap` and `yarn test:unit`, do the tests pass? It's not passing for me. Also, you pushed us a yarn.lock file referencing your artifactory so I was unable to test with exactly what you had.

@schuchard  oh no, sorry. Did not see this. This is wrong. Sorry seeing this being merged. It must be rolled back..

**Pull request merged**